net/dnsviz/Portfile

```diff
@@ -20,11 +20,14 @@ long_description    DNSViz is a tool suite for measurement, diagnostic, and \
                     visualization of Domain Name System (DNS) behavior, \
                     including its security extensions (DNSSEC).
 
-checksums           rmd160 d996637d61539e778cf4035f5c7afefcc03a47ad \
-                    sha256 6f38f3d71b2b9ca3f4cffb003c828574d0c413bcc5112bb52921fa9db4e69259 \
-                    size 388653
+checksums           rmd160 f48389ae74f642a3b25a0690c9760bffcfa0c236 \
+                    sha256 8e2c4d0636296acf704f7eca1ca8fea98b022c920c5517b39dfdc982ce685cd3 \
+                    size 404126
 
-python.default_version  38
+
+python.pep517           no
+python.versions         310 311 312
```

Comment:
this will not do anything for ports that are not named "py-<portname>", where it would create subports. Please remove.

Reply:
Same with `python.default_version`?

Reply:
Fixed in [218df0c](https://github.com/macports/macports-ports/pull/24833/commits/218df0cb93fbfda1ad846592f29fa1b6049ff7a0).

Reply:
no "python.default_version" should be set to "312" here. It will use the default from the "python" PortGroup if not set (which is currently 312 as well) but we don't want ports to suddenly change when that default is updated (which happens on January 1st). So to be specific on which Python version to use we'll always want to set it in the Portfile.